jekyll/_cci2/server-3-whats-new.adoc

```diff
@@ -18,6 +18,12 @@ Server 3.x includes the latest CircleCI features, such as orbs, scheduled workfl
 
 toc::[]
 
+== Release 3.4.8
+
```

Comment:
=== Upgrade notes

Before upgrading to v3.4.x, delete the deployment named `circleci-server-kube-state-metrics`:

. Run `kubectl delete deployment/circleci-server-kube-state-metrics --cascade=orphan`
. Deploy the app from the KOTS Admin Console